<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportConfigDialog</class>
 <widget class="QDialog" name="ReportConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>299</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>284</width>
    <height>299</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Файл отчета</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resfile/src.qrc">
    <normaloff>:/svg/document.svg</normaloff>:/svg/document.svg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Для пользователей группы:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QLineEdit" name="FilePathEdit"/>
   </item>
   <item row="11" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QTextEdit" name="InfoEdit"/>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QLineEdit" name="NameEdit"/>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Файл</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QComboBox" name="comboBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QToolButton" name="BrowsBt">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resfile/src.qrc">
       <normaloff>:/svg/document.svg</normaloff>:/svg/document.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="YesBt">
     <property name="text">
      <string>Применить</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="3">
    <widget class="QPushButton" name="CancelBt">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>YesBt</tabstop>
  <tabstop>CancelBt</tabstop>
  <tabstop>BrowsBt</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>NameEdit</tabstop>
  <tabstop>InfoEdit</tabstop>
  <tabstop>FilePathEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../resfile/src.qrc"/>
 </resources>
 <connections/>
</ui>
